pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -81,6 +82,9 @@ func (s *NodeController) Run(period time.Duration, syncNodeList bool) {
 	// Register intial set of nodes with their status set.
 	var nodes *api.NodeList
 	var err error
+
+	record.StartRecording(s.kubeClient.Events(""), api.EventSource{Component: "nodecontroller"})
```

Comment:
Oh, can we do this?  Run() will return immediately (just spawns some threads).  Is it another go tricks?